__tests__/Unit/Components/Tasks/TaskDetails.test.tsx

```diff
@@ -659,3 +644,65 @@ describe('TaskDependency', () => {
         });
     });
 });
+
+describe('Details component', () => {
+    it('Renders the detail type and value correctly', () => {
+        render(<Details detailType="Assignee" value="John Doe" />);
+        const detailType = screen.getByText('Assignee:');
+        const detailValue = screen.getByText('John Doe');
+        expect(detailType).toBeInTheDocument();
+        expect(detailValue).toBeInTheDocument();
+    });
+
+    it('Renders a GitHub link correctly', () => {
+        const githubIssueUrl = 'https://github.com/example/repo/issues/123';
+        render(<Details detailType="Link" value={githubIssueUrl} />);
+        const link = screen.getByRole('link', { name: /Open GitHub Issue/i });
+        expect(link).toBeInTheDocument();
+        expect(link).toHaveAttribute('href', githubIssueUrl);
+    });
+
+    it('Does not display tooltip on hover when timestamp is not provided', async () => {
+        render(<Details detailType="Started On" value="N/A" />);
+        const detailValue = screen.getByText('Started:');
+        fireEvent.mouseOver(detailValue);
+        const tooltip = screen.queryByText('N/A', { selector: '.tooltip' });
+        expect(tooltip).not.toBeInTheDocument();
+    });
+
+    it('Renders an extension request icon for Ends On with URL', () => {
+        const url = 'https://example.com';
```

Comment:
not a right way to test this feature.
can you bring back the implemented earlier?
![Uploading image.png…]()

Reply:
I have re-written all of these tests, after my implementation all of these tests were failing, so for that reason, I re-write all of these tests according to changes.